lib/circuitbox/faraday_middleware.rb

```diff
@@ -47,6 +47,14 @@ def identifier
 
     private
 
+    def successful_response?(faraday_response)
+      if opts[:error_response]
+        !opts[:error_response].call(faraday_response)
```

Comment:
Do you mind changing this to success ... I'm not a big fan of negative logic it is most likely confusing... and since the function is called success already we can stick with just that (also consider renaming to success?)